CMakeLists.txt

```diff
@@ -797,6 +797,16 @@ if(AVIF_BUILD_APPS)
             LIBRARY DESTINATION "${CMAKE_INSTALL_LIBDIR}"
         )
     endif()
+
+    if(AVIF_ENABLE_EXPERIMENTAL_GAIN_MAP)
+        enable_language(CXX)
```

Comment:
Maybe we should merge all the `if`s at the beginning to set C++ parameters.
This would prevent us from making mistakes when setting the standard to a version somewhere and to a different version somewhere else.

Reply:
Done by vrabaud